modules/nf-core/anota2seq/anota2seqrun/main.nf

```diff
@@ -33,5 +33,7 @@ process ANOTA2SEQ_ANOTA2SEQRUN {
     task.ext.when == null || task.ext.when
 
     script:
-    template 'anota2seqrun.r'
+    """
+    Rscript ${projectDir}/modules/nf-core/anota2seq/anota2seqrun/templates/anota2seqrun.r --output_prefix ${task.ext.prefix ?: meta.id} --sample_treatment_col ${sample_treatment_col} --reference_level ${reference} --target_level ${target} --sample_file ${samplesheet} --count_file ${counts}
```

Comment:
You can't do this, it won't work on cloud providers. Why are you trying to move away from the use of the `template`?